go/vt/sqlparser/ast_rewriting.go

```diff
@@ -32,10 +34,109 @@ type RewriteASTResult struct {
 	AST Statement // The rewritten AST
 }
 
+type ReservedVars struct {
```

Comment:
comments for the public members in this file would be ⭐

Reply:
Done!